src/rpcq.lisp

```diff
@@ -403,8 +401,7 @@ CoreMessages.~A = _deprecated_property(~:*~A)
 
 (defun deserialize (payload)
   "Deserialize the object(s) encoded in PAYLOAD (string or stream)."
```

Comment:
currently the function only decodes byte arrays. Stream support is free if we add `decode-stream` as a possibility